internal/common/common.go

```diff
@@ -190,3 +193,51 @@ func IsSliceNonEmpty(arg interface{}) bool {
 	}
 	return res
 }
+
+func getUnifiedNTPSources(db *gorm.DB, clusterID strfmt.UUID) (string, error) {
```

Comment:
@avishayt do we need this code? if i remember correctly we said that it is user (devops) responsibility to define and configure NTP sources

Reply:
Yes, we said that the NTP sources for the cluster will be the union of the NTP sources from all relevant InfraEnvs.